packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx

```diff
@@ -233,6 +236,20 @@ const AddTilesToDashboardModal: FC<AddTilesToDashboardModalProps> = ({
                                 placeholder="eg. KPI dashboard"
                             />
                         </FormGroup>
+                        <FormGroup
+                            label="Dashboard description"
+                            labelFor="chart-description"
+                        >
+                            <InputGroup
+                                id="chart-description"
+                                type="text"
+                                value={newDashboardDescription}
```

Comment:
non-blocking, but was wondering how this `newDashboardDescription` is being used after user hits submit?

Reply:
good catch! forgot to add it to the mutation